index.html

```diff
@@ -257,7 +258,7 @@
             FeatureLayer, Point, SimpleMarkerSymbol, SimpleLineSymbol, SimpleFillSymbol, UniqueValueRenderer,
             SimpleRenderer,
             Graphic, GraphicsLayer, Query, Basemaps, Search, Legend, LayerList,
-            Print, ScaleBar, deliveryTools) {
+            Print, ScaleBar, deliveryTools, Config) {
```

Comment:
Would `config` rather than `Config` make more sense here since this is an instance and not a class definition?